### ISSUE TITLE
Enhancement: check task in Obsidian when sync'd to TickTick

### ISSUE THREAD

**Issue by ms3056** [thesamim/TickTickSync#102]
Hi - I would like to create tasks in my notes as they occur - but as soon as they are sync'd to TickTick I would like the tasks in Obsidian to be checked as done. I truly want a one-way sync Obsidian → TickTick. 

Thank you for making this plugin.

**Comment by thesamim** [comment]
If tasks are marked done in Obsidian, they are marked done in TickTick. Is that really what you want?

**Comment by LemurTech** [comment]
I don't understand the desired workflow, here. @ms3056 you want the tasks in Obsidian to be marked 'done' as soon as they sync?

**Comment by ms3056** [comment]
Yes - exactly. The workflow is capture in Obsidian and manage in TickTick. As soon as the task is synchronized to TickTick then cross it off in Obaidian. This is a "one way" sync for me. 

I have seen other solutions that do this. I manage ALL tasks in TickTick - I just want to capture some of them in Obsidian - I'll never look at the tasks in Obsidian again.
